### ISSUE TITLE
Error when trying to update support libraries to 28.0.0.2

### ISSUE THREAD

**Issue by adrianknight89** [xamarin/AndroidSupportComponents#195]
### Xamarin.Android Version (eg: 6.0):
9.4.0.51

### Operating System & Version (eg: Mac OSX 10.11):
Windows 10

### Support Libraries Version (eg: 23.3.0):
28.0.0.2

### Describe your Issue:
I have a Xamarin Forms project and want to update support libraries from 28.0.0.1 to 28.0.0.2. Unfortunately, I'm getting the dreaded `nu1107 version conflict detected` error. I decided to open up the .csproj file and manually change the versions to .2.

Here's the error I see in VS `Error List` window when I try to compile:

```
The "DesignTimeBuild" parameter is not supported by the "VerifyVersionsTask" task. Verify the parameter exists on the task, and it is a settable public instance property.	MyProject.Android			

The "VerifyVersionsTask" task could not be initialized with its input parameters.	MyProject.Android	
```

`warning MSB3277: Found conflicts between different versions of "System.Net.Http" that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed.`

**Comment by Jared-Aus** [comment]
Same issue for me. The usual cleans, rebuilds and restarts didn't help. I had to roll back to 28.0.0.1 to keep working.

**Comment by jorgenstorlie** [comment]
Same for me, Xamarin.Android

**Comment by TomBruyneel** [comment]
same problem, trying to update via the package manager makes me install every single support library one by one

**Comment by Nullstr1ng** [comment]
same here... went back to 28.0.0.1 

git restore saved me from killing my self . LOL

**Comment by PouryaNoufallah96** [comment]
sameeeeeeee

**Comment by wuhuangjia** [comment]
VS 2019 Mac + 1

**Comment by scervellera** [comment]
VS 2019 Win +1

**Comment by ruattolo** [comment]
sameeeeeeee

**Comment by nacompllo** [comment]
VS 2017 Win + 1

**Comment by alexandrfalyov** [comment]
You have Jar libraries, common.jar, that have the identical name with inconsistent file contents. Please make sure to remove any conflicting libraries in EmbeddedJar, InputJar and AndroidJavaLibrary

**Comment by MagicAndre1981** [comment]
@Redth 

[28.0.0.3 is tagged](https://github.com/xamarin/AndroidSupportComponents/releases/tag/28.0.0.3) when will it be available?

**Comment by MagicAndre1981** [comment]
ok, [28.0.0.3 is up](https://www.nuget.org/packages/Xamarin.Android.Support.v4/28.0.0.3), but still unlisted. I modified the csproj/packages.config on my own, restored the new nugets and app compiles now fine.

**Comment by adrianknight89** [comment]
28.0.0.3 seems to be working without issues now. @Redth Do you want this ticket closed? Not sure if the fix was included in 28.0.0.3 or the functionality that caused it was excluded.

**Comment by MagicAndre1981** [comment]
> Not sure if the fix was included in 28.0.0.3 or the functionality that caused it was excluded.

yes, [it includes a fix](https://github.com/xamarin/AndroidSupportComponents/commit/1d6e777f25b6174c37643adeabe8d4c9f4bc4234) and you can close it as issue creator, too.
